src/com/palmergames/bukkit/towny/war/common/townruin/TownRuinUtil.java

```diff
@@ -47,7 +50,9 @@ public static boolean isPlayersTownRuined(Player player) {
 			if(resident.hasTown() && resident.getTown().isRuined())
 				return true;
 			
-		} catch (NotRegisteredException ignored) {}
+		} catch (NotRegisteredException ignored) {
+			// Ignored - Maybe add to a debug logger later?
```

Comment:
Because the getTown() is hidden behind a hasTown() we know we can ignore this.

Reply:
While this is true, there's also the possibility of `getResident()` throwing the NotRegisteredException. As this is a public method, it's accessible to other plugins, so actually giving them something to work with is probably the better course of action, is it not?

Reply:
That's valid, I believe this would also end up being sorted out with @silverwolfg11 's resident UUID map PR as well, depending on what was accepted first.